AEPIdentity/Sources/IdentityHitProcessor.swift

```diff
@@ -15,7 +15,6 @@ import Foundation
 class IdentityHitProcessor: HitProcessing {
     private let LOG_TAG = "IdentityHitProcessor"
 
-    let retryInterval = TimeInterval(30)
```

Comment:
It would be nice to keep both, so an extension can provide a retryInterval per entity if needed and also override the default retryInterval for all entities. The retry per entity should have priority if both of them are provided.

Reply:
I'm not a fan of providing both. I find it a bit confusing to implement both IMO. If an implementer would like a default they can handle that within the delegate function. In essence, I'd prefer to have a single source of truth for the retry interval rather than having two, then determine priority.